flow/flow.proto

```diff
@@ -106,6 +117,9 @@ message Flow {
 /* Total amount of data for the whole flow duration */
 	FlowMetric Metric = 32;
 
+/* Floe metric specific to the TCP Protocol only and optional */
+  TCPMetric TCPFlowMetric = 38;
```

Comment:
NodeTID is 33, I think it is impossible in the same struct to have two elements with the same ID